src/app/app.component.ts

```diff
@@ -1,10 +1,19 @@
 import { Component } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { InfoDialogComponent } from './modules/info-dialog/info-dialog.component';
 
 @Component({
   selector: 'ccf-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
-  title = 'ccf-ui';
+  constructor(public dialog: MatDialog) {}
+
+  launchInfoDialog():void {
```

Comment:
This needs to happen lower down, either in ccf-header or ideally in a component like `ccf-info` which contains the button and the dialog popup logic.

Reply:
I pushed new changes to the branch, where I created a ccf-info-button that wraps up all the logic.